### DISCUSSION TITLE
Add the booster vaccines given on the child immunization card page

### DISCUSSION THREAD

**Discussion by ellykits** [opensrp/opensrp-client-path-zeir#102] [Q&A]
@AnnieMungai I wanted to know whether this is a new requirement or not. The booster vaccines implementation was based on private sector (dynamic vaccination). We had the private sector vaccines shown only on the history page. 

The question above references this issue https://github.com/OpenSRP/opensrp-client-path-zeir/issues/74

**Comment by AnnieMungai** [comment]
@ellykits this is a client request for an enhancement. The reasoning was that the HW should be able to see all the vaccines given to the child at one place. It is more like an enhancement to the initial dynamic vaccines feature. 
What I would be concerned about is:
1) Is this something that is do-able?
2) What will be the LOE on it?

**Reply by ellykits** [reply]
I see. There are however some concerns:

The vaccines we show on the immunization card are based on immunization schedules and prompt actions from the user of the app.
If we are to do the same for the Booster vaccines, what are we suppose to show in place? and what should be the resulting action?

**Reply by AnnieMungai** [reply]
@ellykits Can I pull @ndegwamartin into this discussion? He is the specialist on child register. 
So the question goes back to: If we were to add the Booster vaccines on the child immunization card, is it possible? Are we able to add a segment where additional vaccines can be seen by the user on the card? In my head I thought that would be a slight change in UI or something. So we only see the vaccines that were given and when they were given. No action is required from the user or if there is an action, then it would be only for editing - I believe this window is usually like 30min or so.

**Reply by ndegwamartin** [reply]
@AnnieMungai how do these 'booster vaccines' get into the OpenSRP system? i.e. for them to then be rendered on the immunization page?

**Reply by ellykits** [reply]
We add the booster vaccines from a JSON form; saved as  **booster_vaccine** event. They are not scheduled.

**Comment by ellykits** [comment]
The solution to this was not to include the booster vaccines on the vaccine schedules page but implement functionality for editing these booster vaccines like the others.

**Reply by AnnieMungai** [reply]
@ellykits does this mean this is the implementation that will work best with the minimum LOE?
